opm/parser/eclipse/EclipseState/Tables/SimpleTable.hpp

```diff
@@ -60,11 +60,15 @@ namespace Opm {
          */
         double evaluate(const std::string& columnName, double xPos) const;
 
+        /// throws std::invalid_argument if jf != m_jfunc
+        void assertJFuncPressure(const bool jf) const;
```

Comment:
to all of your questions: yep, since IMO it makes about zero sense for e.g., `PvtoTable`. alternatively, you can add a `SatfuncSimpleTable` but I think adding this to all satfunc tables individually is better because this is ECL and it might be that this JFUNC handling does apply to all of them.